sites/all/modules/custom/jdt_user_notes/jdt_user_notes.module

```diff
@@ -5,25 +5,199 @@
  */
 
 /**
- * Implements hook_permission() for jdt_user_notes().
+ * Include CRUD file.
+ */
+include_once 'jdt_user_notes.crud.inc';
+
+/**
+ * Implements hook_permission().
  *
  * Controls permissions.
  */
 function jdt_user_notes_permission() {
   $permissions = array(
-    'creating notes' => array(
+    'create notes' => array(
       'title' => t('Add a Note'),
     ),
-    'reading notes' => array(
+    'read notes' => array(
       'title' => t('Read a Note'),
     ),
-    'updating notes' => array(
+    'update notes' => array(
       'title' => t('Edit a Note'),
     ),
-    'deleting notes' => array(
+    'delete notes' => array(
       'title' => t('Delete a Note'),
     ),
   );
 
   return $permissions;
 }
+
+/**
+ * Implements hook_menu().
+ */
+function jdt_user_notes_menu() {
+  $items = array();
+
+  $items['jdt_user_notes/add/%'] = array(
+    'title' => 'Add a Note',
+    'page callback' => 'jdt_user_notes_add',
+    'page arguments' => array(2),
+    'access arguments' => array('create notes'),
+  );
+
+  $items['jdt_user_notes/edit/%/%'] = array(
+    'title' => 'Edit a Note',
+    'page callback' => 'jdt_user_notes_edit',
+    'page arguments' => array(2, 3),
+    'access arguments' => array('update notes'),
+  );
+
+  return $items;
+}
+
+/**
+ * Custom handler for jdt_user_notes_add().
+ */
+function jdt_user_notes_add($article_nid) {
+  $note_id = '';
+  $form = drupal_get_form('jdt_user_notes_form', $note_id, $article_nid);
+  return $form;
+}
+
+/**
+ * Submits data for jdt_user_notes_add().
+ */
+function jdt_user_notes_add_submit($form, &$form_state) {
+  global $user;
+
+  if (strlen($form_state['input']['note_title']) >= 255) {
+    drupal_set_message(t('Error: The note title is longer than 255 characters. 
```

Comment:
Use the second parameter: `drupal_set_message(t('An error occurred and processing did not complete.'), 'error');`